packages/cli/src/options/beaconNodeOptions/execution.ts

```diff
@@ -41,4 +45,20 @@ export const options: ICliCommandOptions<ExecutionEngineArgs> = {
     type: "string",
     group: "execution",
   },
+
+  "retry.attempts": {
+    description: "Number of retry attempts when calling execution engine API",
+    type: "number",
+    defaultDescription:
+      defaultOptions.executionEngine.mode === "http" ? String(defaultOptions.executionEngine.retryAttempts) : "1",
+    group: "execution",
+  },
+
+  "retry.delay": {
+    description: "Delay time between retries when retrying calls to the execution engine API",
+    type: "number",
+    defaultDescription:
+      defaultOptions.executionEngine.mode === "http" ? String(defaultOptions.executionEngine.retryDelay) : "0",
```

Comment:
do we really need to tailor make it for http? even if there is for example a non http mode ever (like ws) this would stay the same i believe.

Reply:
This was done so as to make it uniform with already existing `execution.urls` and `execution.timeout` options which are tailor made for http.

Reply:
I would just do
```
defaultDescription: String(defaultOptions.executionEngine.retryDelay)
```

Reply:
i think typescript isn't happy otherwise, mixes it up with mock options, so i guess this is alright!